test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -732,6 +735,93 @@ func BenchmarkPerfScheduling(b *testing.B) {
 	}
 }
 
+var testSchedulingLabelFilter = flag.String("test-scheduling-label-filter", "integration-test", "comma-separated list of labels which a testcase must have (no prefix or +) or must not have (-)")
+
+func TestScheduling(t *testing.T) {
+	testCases, err := getTestCases(configFile)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if err = validateTestCases(testCases); err != nil {
+		t.Fatal(err)
+	}
+
+	// Check for leaks at the very end.
+	framework.GoleakCheck(t)
+
+	// All unit test case share the same etcd.
```

Comment:
it's actually integration test, and I am -1 for those integration test to share the same etcd.

Reply:
Agreed, "unit tests" is a leftover and needs to be removed. I don't agree that integration tests shouldn't share the same etcd. Most of them do, via https://github.com/kubernetes/kubernetes/blob/18d05b646d09b2971dc5400bc288062b0414e8cf/test/integration/framework/etcd.go#L186-L222